documentation/src/main/wiki/HandlebarsKnot.md

```diff
@@ -48,6 +48,25 @@ Handlebars Knot uses data from Fragment Context and applies it to Fragment Conte
 ```
 Finally Fragment Content is replaced with merged result.
 
+## Interpolation symbol
+By default Handlebars engine uses `{{` and `}}` symbols as delimiters of variables, expressions, etc.
+However, since the Knot.x can be used to generate markup on server side, it might also consists of markup to be interpreted by the front end engine, such as Angular.
```

Comment:
```
By default Handlebars engine uses `{{` and `}}` symbols as delimiters of variables, expressions, etc.
However, since the Knot.x can be used to generate markup on server side, it might also consists of markup to be interpreted by the front end engine, such as Angular.
```
Just _Knot.x_, _the_ is superfluous. I understand you mean Knot.x as a tool, not a specific instance of Knot.x
As a matter of fact, the article _the_ could be used in the line before, when referring to _the Handlebars engine_ (that Knot.x uses).

How about:
```
By default , the Handlebars engine uses `{{` and `}}` symbols as tag delimiters.
However, while Knot.x can be used to generate markup on the server side, the very same page might also contain templates intended for client-side processing. This is often the case when frameworks like Angular or Vue.js are used.
```

Reply:
Thanks for your comments. Fixed